### ISSUE TITLE
Blank page after upgrade from 8.2 to 9.0 - found work around

### ISSUE THREAD

**Issue by Rocologo** [owncloud/core#23616]
### Steps to reproduce
1. Upgraded 8.2 version to 9.0
2. After upgrade, the Personal Page and the Admin page did not work. They both showed a blank page. 
3. I google the problem and found that it properly was one of the apps which was not 9.0 ready.
4. I therefor disabled all apps, which made the Personal and Admin pages work again.
5. I enabled the apps one after one and found that it was " External storage support 0.5.2" which caused the problem.
6. in the owncloud.log I could see that it was line 268 in google.php which failed.
7. I google that problem and found this [solution](http://stackoverflow.com/questions/1075534/cant-use-method-return-value-in-write-context) 
8. So I change the line from 
"if (empty($child->getFileExtension())"
to
"if ($child->getFileExtension()"
and this fixed the problem. I could now enable "External storage support 0.5.2" and the Personal Page + Admin Page did work again.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

### Server configuration
**Operating system**:

**Web server:**

**Database:**

**PHP version:**

**ownCloud version:** (see ownCloud admin page)

**Updated from an older ownCloud or fresh install:**

**Where did you install ownCloud from:**

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your ownCloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by PVince81** [comment]
Indeed some older PHP versions don't like that construct.

So did you have a GDrive external storage then ?

**Comment by PVince81** [comment]
From what I see this is already fixed in v9.0.1

**Comment by RealRancor** [comment]
Yes, should be a duplicate of https://github.com/owncloud/core/issues/22960 on PHP 5.4 (the used PHP version of @Rocologo is missing. Hinthint: Please always fill out the issue template completely).

Fix is in https://github.com/owncloud/core/pull/22961 which gets shipped with 9.0.1

**Comment by Rocologo** [comment]
Sorry for not filling out the template. I will of cause give you all the info you need. 

1) I don't have any connections to Google drive. I have only connections to my local storage (Samba). 

2) I can confirm that I have "5.4.45-1~dotdeb+7.1" installed on my server. Im using "Openmediavault 2.x" which does not support the never PHP versions.
